Composer/packages/client/src/pages/diagnostics/DiagnosticsTable.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { useMemo, useState } from 'react';
+import formatMessage from 'format-message';
+import { RouteComponentProps } from '@reach/router';
+import { useRecoilValue } from 'recoil';
+import { css } from '@emotion/core';
+
+import { Pagination } from '../../components/Pagination';
+import { diagnosticsSelectorFamily } from '../../recoilModel/selectors/diagnosticsPageSelector';
+import { DiagnosticList } from '../design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticList';
+
+import { IDiagnosticInfo } from './types';
+
+// -------------------- Styles -------------------- //
+
+const listRoot = css`
+  position: relative;
+  overflow-y: auto;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+const tableView = css`
+  position: relative;
+  flex-grow: 1;
+`;
+
+// -------------------- Diagnosticist -------------------- //
+export interface IDiagnosticListProps extends RouteComponentProps {
+  skillId?: string;
+  showType: string;
+  onItemClick: (item: IDiagnosticInfo) => void;
+}
+
+const itemCount = 10;
+
+export const DiagnosticsTable: React.FC<IDiagnosticListProps> = (props) => {
+  const { onItemClick, skillId = '', showType } = props;
```

Comment:
I think the prop the table needs to accept is a project id. Not a skill id. It looks like you are passing projectId ?? skillId from the Diagnostics.tsx

Reply:
yes it should be projectId. I cleared all skillId concepts in Diagnostics.